boshio/http_client_test.go

```diff
@@ -1,44 +1,20 @@
 package boshio_test
 
 import (
-	"errors"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"strings"
 	"time"
 
 	"github.com/concourse/bosh-io-stemcell-resource/boshio"
+	"github.com/concourse/bosh-io-stemcell-resource/fakes"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-type tempError struct {
```

Comment:
Can we bring these back? I see that the test got a bit cleaner from the object perspective but I want to not add additional time to the test suite. The faster the run the better (at least as far as I am concerned).